### PR TITLE
add base-styles

### PR DESCRIPTION
**Pull request by Alisa-Bogatyryova**

### CODE REVIEW

File: src/sass/base/_base-styles.scss [hunk 1]
```diff
@@ -0,0 +1,26 @@
+body {
+  font-family: $main-font, sans-serif;
+}
+html {
+  box-sizing: border-box;
```
Comment (izachok) [inline src/sass/base/_base-styles.scss]: we don't need to use box-sizing with normalize

File: src/sass/base/_base-styles.scss [hunk 2]
```diff
@@ -0,0 +1,26 @@
+body {
```
Comment (izachok) [inline src/sass/base/_base-styles.scss:1]: could you please merge these styles with styles in _common.scss and leave only one file in project

**Review by izachok — CHANGES_REQUESTED** [review]

File: src/sass/base/_base-styles.scss (hunk 1 above)
Comment (Alisa-Bogatyryova) [reply src/sass/base/_base-styles.scss]: исправлено

File: src/sass/base/_base-styles.scss (hunk 2 above)
Comment (Alisa-Bogatyryova) [reply src/sass/base/_base-styles.scss:1]: перенесено, _common.scss удален

**Review by izachok — APPROVED** [review]

**Pull request merged**